package.json

```diff
@@ -83,15 +82,6 @@
     "socket.io-client": "^1.4.6",
     "style-loader": "^0.13.0",
     "url-loader": "^0.5.6",
-    "victory": "^0.1.3",
-    "victory-axis": "^1.5.7",
-    "victory-bar": "^2.5.1",
-    "victory-chart": "^1.2.1",
-    "victory-line": "^0.6.0",
-    "victory-pie": "0.0.1",
-    "webpack": "^1.12.13",
-    "webpack-dev-middleware": "^1.6.1",
-    "webpack-hot-middleware": "^2.10.0",
```

Comment:
oh, I see. for future lookers, dan just removed the stuff from `dependencies` because we already have it in `devDependencies`